Doxyfile

```diff
@@ -32,19 +32,19 @@ DOXYFILE_ENCODING      = UTF-8
 # title of most generated pages and in a few other places.
 # The default value is: My Project.
 
-PROJECT_NAME           = "Graphene"
+PROJECT_NAME           = "Bitshares"
```

Comment:
I think it's better to change it to BitShares-Core

Reply:
done,